src/app/modules/main/stickers/models/sticker_list.nim

```diff
@@ -0,0 +1,60 @@
+import NimQml, Tables, sequtils
+import status/chat/stickers
+import status/types/[sticker]
+import ../../../../../app_service/service/stickers/dto/stickers as stickers_dto
```

Comment:
@jrainville models should work with view items, don't use dto, a purpose of dto is not this one.